### ISSUE TITLE
Is there a way to insert a table using struct

### ISSUE THREAD

**Issue by jianmaikj** [jackc/pgx#1135]
Is there a way to insert a table using struct

**Comment by sovitacharya** [comment]
Yes there is a way by which you can insert data to the table using struct but still you would be requiring each column allocation to each variable of the struct.
